Tamil/Vivekanandha/ponmozhigal.md

```diff
@@ -5,4 +5,7 @@
 
 "???? ??????? ??????? ???????" ????? ????. ?? ?????????? ????????? ???????? ???????? ??????????? ?????????."
 
-"???? ??????; ??????? ?????!"
\ No newline at end of file
+"???? ??????; ??????? ?????!"
+
+அன்பாயிரு!
+எல்லோரிடமும் அன்பாயிரு, துன்பப்படுபவர்களிடம் பரிவுகொள். எல்லா உயிர்களையும் நேசி. யார்மீதும் பொறாமைப் படாதே. பிறரது குற்றங்களைக் காணாதே. -சுவாமி விவேகானந்தர்
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.